packages/@aws-cdk/aws-iam/test/policy-document.test.ts

```diff
@@ -91,22 +93,26 @@ describe('IAM policy document', () => {
     expect(() => {
       new PolicyStatement({
         notActions: ['service:action', '*', 'service:acti*', 'in:val:id'],
+        resources: ['*'],
       });
     }).toThrow(/Action 'in:val:id' is invalid/);
   });
 
   test('Cannot combine Resources and NotResources', () => {
     expect(() => {
       new PolicyStatement({
+        actions: ['abc:def'],
         resources: ['abc'],
-        notResources: ['def'],
+        notResources: ['abc'],
```

Comment:
You should not have needed to change this line. If you have, something is wrong.